pkg/services/store/entity/entity.proto

```diff
@@ -5,12 +5,11 @@ option go_package = "github.com/grafana/grafana/pkg/services/store/entity";
 
 // The canonical entity/document data -- this represents the raw bytes and storage level metadata
 message Entity {
-
   // Globally unique ID set by the system.  This can not be set explicitly
   string guid = 1;
 
-  // The resourceVersion -- it will change whenever anythign on the object is saved
-  string version = 2;
+  // The resourceVersion -- a snowflake id
+  int64 resourceVersion = 2;
```

Comment:
let's be consistent with naming, everything else in the proto and database columns uses snake_case